flycheck.el

```diff
@@ -6920,6 +6920,41 @@ See URL `http://www.erlang.org/'."
    (error line-start (file-name) ":" line ": " (message) line-end))
   :modes erlang-mode)
 
+(defun flycheck-rebar3-project-root (&optional _checker)
+  "Return directory where =rebar.config= is located."
+  (let* ((rebarconfig (locate-dominating-file buffer-file-name "rebar.config"))
```

Comment:
By the way, the name of this variable is wrong, I think.  Afaik `locate-dominating-file` returns the _directory_ containing the file, not the path to the file itself, so the variable doesn't actually contain the path to the rebar configuration file but rather the project directory.

So I guess `project-directory` is the better name here isn't it?